packages/wifi_info_flutter/wifi_info_flutter_platform_interface/lib/src/method_channel_wifi_info_flutter.dart

```diff
@@ -0,0 +1,57 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/services.dart';
+
+import '../wifi_info_flutter_platform_interface.dart';
+import 'utils.dart';
+
+/// An implementation of [WifiInfoFlutterPlatform] that uses method channels.
+class MethodChannelWifiInfoFlutter extends WifiInfoFlutterPlatform {
+  /// The method channel used to interact with the native platform.
+  @visibleForTesting
+  MethodChannel methodChannel =
+      MethodChannel('plugins.flutter.io/wifi_info_flutter');
+
+  @override
+  Future<String> getWifiName() async {
+    String wifiName = await methodChannel.invokeMethod<String>('wifiName');
+    // as Android might return <unknown ssid>, uniforming result
+    // our iOS implementation will return null
+    if (wifiName == '<unknown ssid>') {
```

Comment:
This should probably be done in the Android implementation, to keep the platform interface platform-neutral.

(I know this is just copy paste, just commenting as I'm noticing it)

Reply:
I moved it